### ISSUE TITLE
Issue with "pretty" report not reflecting failed states.

### ISSUE THREAD

**Issue by jmchilton** [galaxyproject/planemo#197]
```
17:39 < g2roboto> [nekrut] jmchilton: another interesting bit:
17:39 < g2roboto> [nekrut] All 5 test(s) executed passed.
17:39 < g2roboto> tophat2[0]: passed
17:39 < g2roboto> tophat2[1]: passed
17:39 < g2roboto> tophat2[2]: failed
17:39 < g2roboto> tophat2[3]: passed
17:39 < g2roboto> tophat2[4]: passed
17:39 < g2roboto> [nekrut] note that is says that all 5 tests passed, but #3 is failed. However, the html file 
                  shown all as apassed
17:40 < jmchilton> ...
17:40 < jmchilton> that seems bad
17:43 < jmchilton> nekrut - is that after an upgrade that fixes the test report or is that the older-style report?
17:43 < g2roboto> [nekrut] planemo 0.11. new style
17:44 < jmchilton> cool - thanks - I'll look into that - wanted to play around with tophat's read group handling 
                   anyway.
```

**Comment by erasche** [comment]
@jmchilton can you get nekrut to retest this now that @bgruening found the thing and it was patched? xref https://github.com/galaxyproject/planemo/issues/363#issuecomment-154820426

**Comment by jmchilton** [comment]
We will call this ancient issue closed with #429.
